internal/pkg/aws/codepipeline/codepipeline.go

```diff
@@ -286,3 +278,19 @@ func (ss *StageState) HumanString() string {
 	}
 	return tree.String()
 }
+
+func addColor(status string) string {
+	const empty = "  -"
+	switch status {
+	case "":
+		return empty
```

Comment:
I wonder if this block should live in its own function since it's not about adding color. What do you think of moving it to: `fmtEmptyStatus`? so that it reads `addColor(fmtEmptyStatus(status))`

Reply:
Is it better practice to keep responsibilities small and separate, and nest the functions like that, or could we rename the existing function ("fmtStatus"?) to be more inclusive and avoid nesting the functions?

Reply:
Mmm good question! I like keeping individual building blocks as separate functions if they can be used by more than one caller. And then create "higher-level" functions like `fmtStatus` that you mention to glue everything up.

In this scenario, since these smaller functions won't be used anywhere but `fmtStatus`, I think having only `fmtStatus` makes sense!

Reply:
Got it. Thanks!